implementation/contracts/system/TBTCToken.sol

```diff
@@ -63,3 +61,6 @@ contract TBTCToken is ERC20Detailed, ERC20, VendingMachineAuthority {
         }
     }
 }
+
+/// @notice Interface of recipient contract for approveAndCall pattern. This is implemented in RedemptionScript.sol
```

Comment:
Same note re: contract vs filename + not implying that's the only place it can be implemented.

As a side note, 80-character wrapping would ideally still apply here.